assets/scripts/segments/Building.jsx

```diff
@@ -122,13 +122,11 @@ class Building extends React.Component {
   handleKeyDown = (event) => {
     if (!this.state.isEditable) return
 
-    const negative = (event.keyCode === KEYS.MINUS) ||
-      (event.keyCode === KEYS.MINUS_ALT) ||
-      (event.keyCode === KEYS.MINUS_KEYPAD)
+    const negative = (event.key === KEYS.MINUS) ||
+      (event.key === KEYS.UNDERSCORE)
```

Comment:
We should be able to remove the underscore check. Previously this was done because the `-` and `_` characters shared the same key, at least on English US keyboards. Now the event returns the actual `-` value, which is what we assume the user wants when they press it.

Reply:
Removed underscore check